azerothcore/azerothcore-wotlk
# fix(DB) A Lost Master missing prerequisite

## Changes Proposed:
-  Add a prerequisite for the quest [[A Lost Master]](https://classic.wowhead.com/quest=986/a-lost-master)


## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/447
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5367



## SOURCE:

https://www.youtube.com/watch?v=7I8Zjd5pUlo
[A Lost Master] is not available at the start of the video, but it is available at the end after turning in [[How Big a Threat?]](https://classic.wowhead.com/quest=985/how-big-a-threat)

## Tests Performed:

- Applied SQL patch
- Tested in-game


## How to Test the Changes:

- `.go c terenthis`
- Verify that he gives [How Big a Threat?] but not [A Lost Master]
- Complete the two quests named [How Big a Threat?]
- Verify that he gives [A Lost Master] now


## Known Issues and TODO List:

- [ ] When turning in [How Big a Threat?], Terenthis should say some lines, and Grimclaw should appear (reported here https://github.com/azerothcore/azerothcore-wotlk/issues/2118 and here https://github.com/chromiecraft/chromiecraft/issues/452), see [the end of the video linked earlier](https://youtu.be/7I8Zjd5pUlo?t=291).


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
